crates/cxx-qt-gen/src/generator/structuring/mod.rs

```diff
@@ -54,6 +71,30 @@ impl<'a> Structures<'a> {
             }
         }
 
+        for method in &cxxqtdata.methods {
+            println!(
+                "Looking for QObject match for method: {:?}",
+                method.name.clone()
+            );
+            if let Some(qobject_ident) = qobjects.iter_mut().find(|qobject| {
+                qobject.declaration.name.rust_unqualified() == &method.qobject_ident
+            }) {
```

Comment:
This looks quite intense to be doing for every method we iterate over all qobjects?

Does this point that we should store the qobjects above as a hash instead by their rust declaration name?

```
let mut qobjects: HashMap<Ident, StructuredQObject>
```

Or should there be a cache of the index or a mut ref (might be very tricky with lifetimes). Eg `HashMap<Ident, usize>` or `HashMap<Ident, &mut StructuredQobject>` so we can get from the qobject ident -> structured qobject easily?